lib/src/main/java/com/thenewboston/data/dto/bankapi/bankdto/BankDTO.kt

```diff
@@ -0,0 +1,18 @@
+package com.thenewboston.data.dto.bankapi.bankdto
+
+import com.google.gson.annotations.SerializedName
+
+data class BankDTO(
+    @SerializedName("account_number")
+    val accountNumber: String,
+    @SerializedName("ip_address")
+    val ipAddress: String,
+    @SerializedName("node_identifier")
+    val nodeIdentifier: String,
+    val port: Int? = null,
+    val protocol: String,
+    val version: String,
+    @SerializedName("default_transaction_fee")
+    val defaultTransactionFee: Double,
```

Comment:
Shoudn`t we use BigDecimal for monetary types ?

Reply:
In this case, bigdecimal will truncate the total decimal values when parsing the JSON response from API, used double as it will keep the original precision. eg- 1.0000000000 from response will be truncated to 1.0 with big decimal